grails-app/services/au/org/ala/volunteer/ProjectService.groovy

```diff
@@ -1147,7 +1198,7 @@ class ProjectService implements EventPublisher {
      * Retrieves background image url
      * @return background image url or null if non existent
      */
-    String getBackgroundImage(Project project) {
+    String getBackgroundImage(Project project, boolean absolute = true) {
```

Comment:
This parameter seems to determine whether it returns an HTTP url or a file system path and not whethere the path is absolute or not? Should probably be renamed?